manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     security.istio.io/tlsMode: {{ index .ObjectMeta.Labels `security.istio.io/tlsMode` | default "istio"  | quote }}
     service.istio.io/canonical-name: {{ index .ObjectMeta.Labels `service.istio.io/canonical-name` | default (index .ObjectMeta.Labels `app.kubernetes.io/name`) | default (index .ObjectMeta.Labels `app`) | default .DeploymentMeta.Name  | quote }}
     service.istio.io/canonical-revision: {{ index .ObjectMeta.Labels `service.istio.io/canonical-revision` | default (index .ObjectMeta.Labels `app.kubernetes.io/version`) | default (index .ObjectMeta.Labels `version`) | default "latest"  | quote }}
-    istio.io/rev: {{ .Revision | default "default" | quote }}
```

Comment:
I think we had discussed putting it in the status annotation? Since we are already planning to add other things like user-agent, etc. Labels are not free - they show up in some `kubectl commands` (developer UX cost) and create indexes in etcd (performance cost).

They are pretty cheap but still seems better to put it in status, since this is really just status.

Reply:
Makes sense, moved to status annotation.